test/extended/networking/services.go

```diff
@@ -0,0 +1,67 @@
+package networking
+
+import (
+	"k8s.io/kubernetes/test/e2e"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("[networking] services", func() {
+	f1 := e2e.NewFramework("net-isolation1")
+	f2 := e2e.NewFramework("net-isolation2")
+
+	It("should allow connections to other pods via service IPs", func() {
+		By("Picking multiple nodes")
+		nodes := getMultipleNodes(f1)
+		node1 := nodes.Items[0]
+		node2 := nodes.Items[1]
+
+		By("Running a webserver")
+		serviceName := "webserver"
+		defer f1.Client.Pods(f1.Namespace.Name).Delete(serviceName, nil)
+		defer f1.Client.Services(f1.Namespace.Name).Delete(serviceName)
+		ip := launchWebserverService(f1, serviceName, node1.Name)
+
+		By("Trying to connect to the service from a pod on the same node")
+		Expect(checkConnectivityToHost(f1, node1.Name, "service-same-node-wget", ip, 10)).To(Succeed())
+
+		By("Trying to connect to the service from a pod on a different node")
+		Expect(checkConnectivityToHost(f1, node2.Name, "service-different-node-wget", ip, 10)).To(Succeed())
+	})
+
+	var expectation string
+	if isMultitenant() {
```

Comment:
Same comment here as the pod isolation check regarding the desirability of defining tests statically.